webcomponents/tool/src/main/frontend/js/conversations/sakai-comment.js

```diff
@@ -67,9 +67,15 @@ export class SakaiComment extends SakaiElement {
 
     return html`
       <options-menu placement="bottom-left">
-        <a slot="trigger" id="options-menu-link-${this.comment.id}" href="javascript:;">
-          <div><sakai-icon type="menu" size="small"></sakai-icon></div>
-        </a>
+        <div slot="trigger">
+          <a href="javascript:;"
+              title="${this.i18n.comment_options_menu_tooltip}"
+              aria-haspopup="true"
+              aria-label="${this.i18n.comment_options_menu_tooltip}">
+            <sakai-icon type="menu" size="small"></sakai-icon>
+          </a>
```

Comment:
this seems fine but when i see href="javascript" i get kind of suspicious.

why not a <button>?

Reply:
I use "javascript:;" all the time as I don't want the page hopping to the top through using a #. What's the better solution?

Reply:
button

Reply:
Alright, I'll switch this to button. What about when things have to be links though? Transparent button?

Reply:
I agree entirely. A "nude" button would do the trick.

Reply:
I don't know if I like that commit I just pushed. It's more !importants and I know we don't want those.